### ISSUE TITLE
Existing style guide resources 

### ISSUE THREAD

**Issue by laurawesley** [wet-boew/wet-boew-styleguide#2]
This repo has the existing [WxT 3.1 style guide](http://wet-boew.github.io/wet-boew-styleguide/guide-en.html) in it. We need to update it for WxT4.0, review it against usability and accessibility heuristics as well as any test results AND add what's missing.

UXWG had started [a GC style guide](http://www.gcpedia.gc.ca/wiki/Government_of_Canada_style_guide_for_web) (internal link) but the timing wasn't right. The project outline, names of interested parties, [Content presentation guidance](http://www.gcpedia.gc.ca/wiki/Government_of_Canada_style_guide_for_web/Content_Presentation_Guidance) (internal link) including the [section on usability](http://www.gcpedia.gc.ca/wiki/Government_of_Canada_style_guide_for_web/Content_brainstorm/Usability_Guidance_for_GC_Style_Guide_Usability) (internal link)  may still be useful. [Departmental style guides](http://www.gcpedia.gc.ca/wiki/GC_Web_Style_Guide/Research) (internal link) for ideas & comparisons are also listed on GCPEDIA.

External style guides we love:
- [Global Experience Language for the BBC](http://www.bbc.co.uk/gel)
- [Bootstrap](http://getbootstrap.com/) -- which we are using in WxT4
- [Foundation](http://foundation.zurb.com/docs/)
- [GitHub style guide](https://github.com/styleguide/)
- [Batman.js](http://batmanjs.org/)
- [Ember.js](http://emberjs.com/)
- [Backbone.js](http://backbonejs.org/)

Other resources that we love:
- [BC Gov's UX Toolkit](http://www2.gov.bc.ca/gov/topic.page?id=46DB9FF151D14B84B86FEE25CD206989)
- [Gov UK's Service Design Manual](https://www.gov.uk/service-manual)

Writing resources on GCPEDIA:
- [Transport Canada](http://www.gcpedia.gc.ca/wiki/Writing_for_the_Web)
- [CanadaInternational](http://www.gcpedia.gc.ca/wiki/CanadaInternational_Web_Network/Training/Web_Writing)

Related issues discussing styles, documentation, formatting etc:
https://github.com/wet-boew/wet-boew/issues/1077 - oldest discussion about WxT documentation 
https://github.com/wet-boew/wet-boew/issues/1918 - using KSS for CSS documentation / living style guide
https://github.com/wet-boew/wet-boew/issues/1275 - WxT4.0 planning thread
https://github.com/wet-boew/wet-boew/issues/2813 - WxT website IA planning

Supposedly, there is some [old thing here](http://www.collectionscanada.gc.ca/webarchives/20071212093255/http:/www.tbs-sct.gc.ca/clf2-nsi2/tb-bo/td-dt/wewd-rdwe-eng.asp#cn_4.0_as) but I can't access it on my antiquated browser (or it may be a broken link, not sure).

@jeresiv @cfarquharson @MaryBethBaker @sagecram et al
